### PR TITLE
ui: add guide modal

### PR DESCRIPTION
**Pull request by harunurhan**

### CODE REVIEW

File: ui/src/reducers/ui.js [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
 
-import { UI_CLOSE_BANNER } from '../actions/actionTypes';
+import {
+  UI_CLOSE_BANNER,
+  UI_CHANGE_GUIDE_MODAL_VISIBILITY,
+} from '../actions/actionTypes';
 
 export const initialState = fromJS({
   closedBannersById: {},
+  guideModalVisibility: null,
```
Comment (drjova) [inline ui/src/reducers/ui.js]: why not just ``true`` ``false``

**Comment by drjova** [comment]
@harunurhan is this the final?

File: ui/src/reducers/ui.js (hunk 1 above)
Comment (harunurhan) [reply ui/src/reducers/ui.js]: null has a meaning, it means it was never displayed before.

**Comment by harunurhan** [comment]
@drjova no waiting for actual step images, will merge immediately when this comes

**Pull request merged**